kommunicate/src/main/java/com/applozic/mobicomkit/channel/service/ChannelClientService.java

```diff
@@ -33,7 +33,7 @@ public class ChannelClientService extends MobiComKitClientService {
     private static final String CHANNEL_INFO_URL = "/rest/ws/group/info";
     // private static final String CHANNEL_SYNC_URL = "/rest/ws/group/list";
     private static final String CHANNEL_SYNC_URL = "/rest/ws/group/v3/list";
-    private static final String CREATE_CHANNEL_URL = "/rest/ws/group/create";
+    private static final String CREATE_CHANNEL_URL = "/rest/ws/group/v2.1/create";
```

Comment:
Are we sure upgrading this API will not cause any other problem?
Request Response payload is same for both?

Reply:
both are same